### PR TITLE
feat: angular package format

### PR DESCRIPTION
**Pull request by yggg**
### Please read and mark the following check list before creating a pull request:

 - [x] I read and followed the [CONTRIBUTING.md](https://github.com/akveo/nebular/blob/master/CONTRIBUTING.md) guide.
 - [x] I read and followed the [New Feature Checklist](https://github.com/akveo/nebular/blob/master/DEV_DOCS.md#new-feature-checklist) guide.
 
 #### Short description of what this resolves:

Adds `esm2015`, `fesm2015`, `fesm5` into packages distributives.

BREAKING CHANGE:
Everything not mentioned in packages `public_api.ts` now private. Direct imports from (`@nebular/theme/*`) will stop working.

### CODE REVIEW

File: scripts/gulp/tasks/change-prefix.ts [hunk 1]
```diff
@@ -6,7 +6,7 @@ import * as replace from 'gulp-replace';
 import * as minimist from 'minimist';
 import { capitalize, dasherize } from '@angular-devkit/core/src/utils/strings';
 
-import { replaceScssWithCss } from './copy-sources';
+// import { replaceScssWithCss } from './copy-sources';
```
Comment (Tibing) [inline scripts/gulp/tasks/change-prefix.ts]: I think that line has to be removed

File: scripts/ci/packages-smoke.sh [hunk 2]
```diff
@@ -17,8 +17,10 @@ packages_smoke() {
   cp -r ../nebular/src/.lib/* node_modules/@nebular
 
   echo "Verifying application build"
-  npm run build --prod --aot
+  npm run build -- --prod --aot
```
[the remark this answers is not in the document]
Comment (yggg) [reply scripts/ci/packages-smoke.sh]: Removed in 3ef9057

File: DEV_DOCS.md [hunk 3]
```diff
@@ -27,6 +27,7 @@
 - add the package into bump-version.ts which bumps package version and its dependencies
 - add the package external dependencies into rollup-config.ts which gives rollup capability build package correctly
 - add the package into bundle.ts which build umd modules for our packages
+- add the package into `JS_PACKAGES` in config.ts which used to add es2015 bundles for our packages
```
[the remark this answers is not in the document]
Comment (yggg) [reply DEV_DOCS.md]: Added in 9d0cebd

File: scripts/gulp/tasks/change-prefix.ts (hunk 1 above)
Comment (yggg) [reply scripts/gulp/tasks/change-prefix.ts]: Removed in c15e9b6

File: scripts/gulp/tasks/change-prefix.ts [hunk 4]
```diff
@@ -218,7 +218,7 @@ task('change-prefix', ['copy-build-dir-and-rename', 'generate-ts-config', 'patch
 
   return stream
     .pipe(dest(BUILD_DIR))
-    .on('end', replaceScssWithCss);
+    // .on('end', replaceScssWithCss);
```
[the remark this answers is not in the document]
Comment (yggg) [reply scripts/gulp/tasks/change-prefix.ts]: Removed in c15e9b6

**Comment by Tibing** [comment]
I personally like the implementation 😄 👍

**Comment by Tibing** [comment]
LGTM

**Comment by yggg** [comment]
Thanks 🙂 
But hopefully we'll switch to bazel and packagr soon.

**Pull request merged**